### DISCUSSION TITLE
Introduce Yourself

### DISCUSSION THREAD

**Discussion by Harshee-Pitroda** [Harshee-Pitroda/Git-Version-Control#3] [General]
This thread is intended for new community members to introduce themselves.

**Comment by Shamanthsunny** [comment]
Hello fellow developers,
I'm Shamanth, Currently pursuing B.Tech 4th year in Mechanical Engineering in MITS College Madanaplle. I am thrilled to part of this sessions . I hope that in these sessions I could grasp the fundaments of Git and GitHub and make use of it for my future career aspects . This is my Github id : Shamanthsunny

**Reply by Harshee-Pitroda** [reply]
Thank you Shamanth for introducing yourself!

**Comment by KONETISUPRIYA** [comment]
I am K SUPRIYA and I am currently studying 3rd year of btech from SIDDHARTH GROUP OF INSTITUTIONS PUTTUR.
Operating system:Windows

**Reply by Harshee-Pitroda** [reply]
Thank you for introducing yourself!

**Comment by UDSravani** [comment]
UD.Sravani
4th year B.Tech student from Madanapalle Institute of Technology and Science, Madanapalle 
Operating system: Windows
GitHub id:- UDSravani
Thank you for giving me this opportunity, excited to explore new things.

**Reply by Harshee-Pitroda** [reply]
Thank you for introducing yourself Sravani

**Comment by adarshba** [comment]
Hello, I'm Adarsh, a sophomore in BE at DSATM Bangalore. I am immensly grateful for such valuable sessions and I would make sure to make good use of this oppurtunity by keeping myself engaged throughout the sessions. Thank you Harshee ma'am for this wonderful oppurtunity.
Here's my github ID: Adarsh79

**Reply by Harshee-Pitroda** [reply]
Thank you Adarsh for introducing yourself

**Comment by harsha86604** [comment]
Harsha - G7 DS
Hello Everyone I am Harshavardhan pursuing BTech 3rd in Sri Venkateswara College of Engineering and technology Chittoor.I am very excited to learn about git and github which helps in developing the projects and contribute to others. I hope this learnings will help me in the IT field

**Reply by Harshee-Pitroda** [reply]
Thank you Harsha for introducing yourself!

**Comment by PoojithaJonnagaddala** [comment]
Hi, I'm Poojitha
Btech final year student from IIIT RGUKT R K Valley.
Iam very much excited to learn about git and GitHub.
Thank you for giving me this opportunity.
Operating System: Ubuntu
GitHub ID: PoojithaJonnagaddala

**Reply by Harshee-Pitroda** [reply]
Thank you for introducing yourself here Poojitha

**Comment by Mamatha81230** [comment]
Hello, I am Mamatha (G15 FSD )  pursuing my Degree 
In Brindavan college at banglore. 
It's really great full to learn new technology . Thank you Harshee ma'am for this wonderful oppurtunity.
My git hub ID : Mamatha81230

**Comment by Cheekala1234** [comment]
Hello
I am C.Ramyasree
B.Tech 4th year from Madanapalle Institute of Technology and Science , Madanapalle
GitHub ID : Cheekala1234
Operating System : Windows
I am really happy and eager to try out new things!

**Comment by lohitha-1123** [comment]
Hello
I am M. Lohitha
B. Tech graduate from R.M.K. College Of engineering and technology, GUMMIDIPOONDI, Chennai
GitHub ID: lohitha-1123
Operating system : Windows
I am very much excited to learn about Git and GitHub
Thank you for giving me this opportunity

**Comment by vamsikrishna1006** [comment]
Hello mam, 
I am C. VAMSI KRISHNA
Currently pursuing B.Tech 3rd year in Electronics and communication engineering in Siddharth Institute of Engineering and Technology, Puttur.
GitHub ID: vamsikrishna1006
Operating system: Windows 11

**Comment by 9392473947** [comment]
hello mam
i am Chaithanya
B-Tech 3rd year CSE
github id: 9392473947
os:windows 11

**Comment by TharaniReddyThugu** [comment]
Hello all..!!
I am Tharani [G26_Python] persuing 3rd year B.Tech in SIDDHARTH INSTITUTE OF ENGINEERING AND TECHNOLOGY,Puttur.
OS: Windows
Excited to be here and ready to learn new things about Git,Github usage.
Thank you harshee mam.
Github ID :TharaniReddyThugu

**Comment by Dineshpundamalli** [comment]
Hello! Iam Dinesh pundamalli
Pursuing Bsc.MECs(3rd year)
MSR Degree College
Kavali 
OS: Windows
GitHub I'd:Dineshpundamalli
Thank u..

**Comment by kallepalli-bhavana** [comment]
Hello mam
I am k. Bhavana 
B. SC graduate from Pragati women's degree College
GitHub ID: kallepalli -bhavana
Operating system : Windows 11 22H2 version 
Thank you

**Comment by AnandRevanthKumar** [comment]
Name: Anand Revanth Kumar
1st year MCA from KMM Institute of PG Studies, Tirupati
Operating system: Windows
GitHub id:- https://github.com/AnandRevanthKumar

Thank you, Waiting  to explore new things.

**Comment by Yaseer-03** [comment]
Name: Alur Mohammed Yaseer
Course: pursuing 3rd B.Tech in computer science
College: St.johns college of engineering and technology,Yerrakota (Kurnool Dist)
OS: WINDOWS 11
GitHub id: https://github.com/Yaseer-03

**Comment by BalijaHari** [comment]
Name : BALIJA HARI
College : Annamacharya institute of technology and scirnce rajampet
Operating system : windows
GitHub Id : https://github.com/BalijaHari

**Comment by datta141** [comment]
Hello Everyone 
I am Dattasai Danta
Cuurrently I'm Pursuing 2nd year M.Sc
Operating System: Windows 11
GitHub id: datta141

**Comment by Srihitha1** [comment]
Hello
I am G.Srihitha
B.Tech 3rd year from Vignan University, Vadlamudi 
GitHub ID : Srihitha1
Operating System : Windows
I am really happy and eager to try out new things!

**Comment by Bhargavi-g28** [comment]
I am Bhargavi,I am pursuing mca 1st year in SV University tirupati .   I am eagerly waiting to know 
about git and GitHub . Thanking you ma'am for giving this opportunity 
Operating system : windows

**Comment by ManishBabu-1** [comment]
Hello everyone,
Name:- K Manish Babu
College:- Mahatma Gandhi Institute of 
                 Technology, Gandipet, Hyderabad.
                 Pursuing 4th year B.Tech in ECE 
                 domain.
Operating System:- Windows
GitHub I'd:- ManishBabu-1

**Comment by Nharika41** [comment]
Hello Everyone,
This is Harika Nagalingam,
Pursuing B.tech 3rd year(AIML) at Vignan University, Guntur.
I'm using Windows 11 operating system on my laptop.
My GitHub I'd: https://github.com/Nharika41

**Comment by Bannu7** [comment]
Hello
I am Y Manvitha
B.Tech 3rd year from Vignan University, Vadlamudi
GitHub ID : Bannu7
Operating System : Windows
I am really happy and eager to try out new things!

**Comment by SHAIKSAMEER00** [comment]
HELLO EVERYONE,
I am SHAIK SAMEER currently pursuing BTECH 4th year in Mechanical Engineering in MITS college, MADANAPALLE. I am very excited to learn  GIT and GITHUB. 
GitHub ID : SHAIKSAMEER00
Operating System : Windows

**Comment by SougataDe1** [comment]
I am SOUGATA DE.
B. Tech(ECE) Graduate from ALIAH UNIVERSITY, KOLKATA
Operating system: windows
My GitHub I'd:- https://github.com/SougataDe1

**Comment by Kckarthik06** [comment]
I am k.chandra karthik
B.tech 4th year student from YSR ENGINEERING COLLEGE OF YOGI VEMANA UNIVERSITY, proddatur
Operating system: windows
My github id : Kckarthik06

**Reply by THE-HACKER7** [reply]
HELLO DEVELOPERS
I'M SAI JAYANTH COMPLETED MCA IN ANDHRA UNIVERSITY. I'm excited know about git hub thank you for giving this opportunity to learn new skills
Operating system: windows11
My git hub id:THE-HACKER7

**Comment by venkatesh057** [comment]
I am T. Venkatesh and I am currently studying 4th year of btech from Sasi Institute of technology and engineering Tadepalligudem 
Operating system:Windows
Github id:https://github.com/venkatesh057

**Comment by diwakar9577** [comment]
Hello everyone
myself Diwakar Allampalli
studying 3rd year AIML in Vignan University
excited to be part of this course
OS : windows 
Github_id:diwakar9577

**Comment by PallaSwetha** [comment]
I am Swetha, pursuing B.Tech final year at AP IIIT RGUKT RKValley in Electronics and Communications. Thank you for providing this opportunity.

**Comment by Nagireddy-18** [comment]
G11_vlsi_frontend 
I am Nagireddy, pursuing B.Tech final year at Kakatiya institute of technology and science, warangal  in Electronics and Communications. Thank you for providing this opportunity.
GitHub I'd: Nagireddy-18.

**Comment by tharun0306** [comment]
Hello everyone,
Name: Mallisetty Sai Tharun
College: CMR Technical Campus,Kandlakoya, Hyderabad.
Persuing B.Tech 4th year in ECE domain.
Operating system: windows 11
Git hub I'd: tharun0306

**Comment by ghost** [comment]
Hello,my name is Deepak.B and I am currently studying in Don Bosco Institute of technology(ECE dept),Bengaluru 
Profile name:- NanuDeepak

**Comment by Nishant-Narvekar** [comment]
Hi, I'm Nishant Narvekar. I have completed my Masters from Parul University and I am currently mentoring students in the IERY AutoCAD domain. I intend to learn Git and GitHub and use it to enhance my skill-set!

**Comment by ranganath-ai** [comment]
Hi, good evening. I'm C.Ranganath and i'm pursuing B.Tech in Mechanical Engineering at Madanapalle institute of technology and science. I completed Data Science and Analytics course in SURE Trust.

**Comment by alekhyae5** [comment]
Hi
I am E.Alekhya
I am pursuing B.Tech final year in CSE-Data Science at Madanapalle Institute of Technology and Science.
I am very exicted to learn about git and github.

**Comment by DABBARAASWANI777** [comment]
Hi
I AM D.ASWANI_G22-Python&ML Basic Applications
I am pursuing B.Tech final year in Electronics and Communication Engineering at RGUKT,R.K.VALLEY.
Thank You for giving this opportunity

**Comment by UmeshChandra-G** [comment]
Hi madam ! I am GORREPATI UMESH CHANDRA.
And I am currently studying btch .3rd year from Siddharth institute of engineering and technology.
Os: windows

**Comment by Nithish-Varma** [comment]
Hello all,
I am Nithish Varma, Currently pursuing B.Tech 4th year in Mechanical Engineering in MITS College Madanapalle. I am soo excited to be a part of this.  Coundn't wait to learn new things in Git and GitHub and try them out.
My Git id is : Nithish-Varma

**Comment by iamsainaresh** [comment]
Hello all, 
I am PADAKANTI SAI NARESH, I am an ECE UG student studying 3rd year at RGUKT-IIIT, RK-VALLEY, I am very glad to be part of this GIT-HUB training session, I am ready to learn new things about GIT and GITHUB and give my level best outputs. 

My Git id is :  iamsainaresh .

**Comment by UmeshChandra-G** [comment]
Hello mam!
My name is G.UMESH CHANDRA
Currently pursuing B.Tech 3rd year in Electronics and communication engineering in Siddharth Institute of Engineering and Technology, Puttur.
GitHub ID: UmeshChandra-G
Operating system: Windows 11

0 replies

**Comment by Thulasiranirg** [comment]
I am R G THULASIRANI
Studying 3rd B.Tech In The stream of ECE In Siddharth Institute of Engineering and Technology, Puttur
I am currently doing G27 Python with Machine Learning in Sure Trust... 
My Laptop Operating System is Windows 11
My Github Id : Thulasiranirg

**Comment by NAMBALAREDDYKEERTHI** [comment]
Hi
I am NAMBALA REDDY KEERTHI 
I am pursuing B.Tech final year in CSE-Data Science at Madanapalle Institute of Technology and Science.
I am very exicted to learn about git and github

**Comment by malleswari-1613** [comment]
Hi
I am Segireddy Naga malleswari
I am pursuing B.Tech final year in the department of vse from RGUKT RKValley and I'm currently doing g-16 FSD
I am very exicted to learn about git and github
Operating system:windows 11
github id- malleswari-1613

**Comment by Ayshakhan01** [comment]
Hey Everyone 
I am T Aysha khan
Currently I'm Pursuing 3rd year B-Tech (CSE)
Operating System: Linux 
GitHub id:https://github.com/Ayshakhan01

**Comment by B-pallavi123** [comment]
Hello everyone,I am B.Pallavi pursuing B.Tech 2nd year in IIIT RGUKT RK Valley. I am currently doing DSA in Python in SURE Trust.
It's really a wonderful oppurtunity to me to learn about Git hub.I am really super excited to learn.
Thank you Harshee mam.
Github_ID:B-pallavi123
Operating_system:UBUNTU

**Comment by arun2022raj** [comment]
Hello everyone, my name is JAKKAM ARUN RAJESH. Currently, I am pursuing my Master of Computer Applications at MITS in Madanapalle. I am also enrolled in the "SQL & Power BI" course at Sure Trust.

I am delighted that Sure Trust offers numerous courses to help students build their professional careers. I feel privileged to be a part of the Sure Trust family, and I am eager to learn more about GitHub.

Thank you for your attention mam.

Best regards,
JAKKAM ARUN RAJESH

**Comment by Edagottuchandana** [comment]
good evening mam, my name is E.Chandana pursuing B.Tech 3nd year in IIIT RGUKT RK Valley. I am currently doing Full stack development in SURE Trust.
It's really a wonderful opportunity for me to learn about Git hub.
Thank you Harshee mam.
My Github Id :Edagottuchandana
Operating-system:UBUNTU

**Comment by poojithakota010** [comment]
Hello everyone., 
I am poojitha kota, pursuing B. Tech in RGUKT. ,
I am currently doing VLSI Front end in sure trust. 
I am really excited to learn GitHub through sure trust.
Github id: poojithakota010
Operating system:windows
  Thank you, harshee mam

**Comment by 2405vaishnavi** [comment]
hello everyone,
I am Vaishnavi Dhare I am Pursuing MCA
Currently I am doing work on java ivery domain
It's really a wonderful opporunity for me to learn about git and github
Operating system - Windows
Github id :https://github.com/2405vaishnavi

**Comment by devchadha-jmi** [comment]
Hi all!
Dev here! I am a BTech final year student at Jamia Millia Islamia, New Delhi
OS: I have a Windows laptop and I use WSL for Linux

PS: Sorry I was stuck at something.. couldn't get involved previously :/

**Comment by DivyasreeTankala** [comment]
Hello Everyone,
My name is Divyasree. I have completed my Engineering from RGUKT Srikakulam.
OS : Windows 10
Git hub ID : DivyasreeTankala
Thank you for providing us this valuable sessions on GitHub. I am very excited to learn things about Git and GitHub

**Comment by NivedithaGunapati** [comment]
Hi Everyone
I am G.Niveditha
Currently I am persuing B.Tech 3rd year
Operating System:Linux(ubuntu)
GthubId: NivedithaGunapati

**Reply by meesalapravas12** [reply]
Hello Everyone,
This is Pravallika Meesala,
Pursuing B.tech 3rd year(CSE) at Velammal Institute of Technology,chennai.
I'm using Windows 11 operating system on my laptop.
My GitHub I'd: https://github.com/meesalapravas12/meesalapravas12

**Comment by RamakrishnaRedddy** [comment]
Hi Everyone
I am ch.Ramakrishna Reddy
Currently I am presuring B.Tech 3rd year
Operating system: windows
Gthubld: Ramakrishna Reddy
